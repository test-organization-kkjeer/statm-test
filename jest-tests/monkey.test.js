describe('monkey-tests', () => {
  it("has a basic snapshot test", () => {
    expect('monkey 234234').toMatchSnapshot();
  });
});
