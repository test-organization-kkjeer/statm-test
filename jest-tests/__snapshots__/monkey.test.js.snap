// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`monkey-tests has a basic snapshot test 1`] = `"monkey"`;
